README.md

```diff
@@ -573,3 +574,175 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+## External libraries
+
+### Prefer native js vs lodash
+
+> Reason: lodash is a great library which contains a lot of useful functions but the modern js has the short identical part of this functionality so some lodash functions are excess alternative.
+
+> Don't forget to check necessary browsers compatibility and turn on polyfills if it's needed.
```

Comment:
A good note, I like it. I guess it would be better to clarify which variants is dangerous and add a link to MDN for every such method. We have to notify a users of this document.

Reply:
I suggest to not create a compatibility map, all developers understand about using different functionality and should think about each method itself depending which browser support is needed for an application